config.toml

```diff
@@ -11,6 +11,6 @@ disableKinds = ["section", "taxonomy", "taxonomyTerm", "RSS", "sitemap", "robots
 # rename when the logo is updated, so we indirectly also invalidate caches
 logo = "images/hypper-logo-header.png"
 githubLink = "https://github.com/rancher-sandbox/hypper"
-docsLink = "https://github.com/rancher-sandbox/hypper/blob/main/docs/SUMMARY.md"
+docsLink = "https://rancher-sandbox.github.io/hypper/html/"
```

Comment:
https://rancher-sandbox.github.io/hypper/ is now the correct link!

Reply:
fixed in [#0877a44](https://github.com/rancher-sandbox/hypper.io/pull/9/commits/0877a446840aa35fcd7839a400652ae5056d6de6)